cdap-watchdog/src/main/java/co/cask/cdap/logging/save/KafkaMessageCallback.java

```diff
@@ -94,7 +95,7 @@ public void onReceived(Iterator<FetchedMessage> messages) {
       } catch (Throwable th) {
         LOG.error("Error processing message at topic {} parition {}",
```

Comment:
Since the exception is not propagated, I think it's good to indicated that in the error message as well, something like "Message ignored due to exception for topic {}, partition {}, message {}"...